cmd/gossamer/account.go

```diff
@@ -164,13 +166,13 @@ func generateKeypair(keytype, datadir string, password []byte) (string, error) {
 	var err error
 	if keytype == crypto.Sr25519Type {
 		// generate sr25519 keys
-		kp, err = crypto.GenerateSr25519Keypair()
+		kp, err = sr25519.GenerateKeypair()
```

Comment:
We should probably update `generateKeypair` to take ` crypto.KeyType`

Reply:
why? it's now part of the subpackage so it's specific to each key implementation